.env

```diff
@@ -1,4 +1,4 @@
 REG=100225593120.dkr.ecr.us-east-1.amazonaws.com
-ALLIANCE_RELEASE
+ALLIANCE_RELEASE=5.3.0
```

Comment:
@christabone @sierra-moxon is this file even used?

Reply:
I'm not sure, it looks like @mluypaert added it in Jan 2021?

Reply:
@christabone @oblodgett Yes, this file is being used by docker compose, so when running the loader locally, though I wouldn't hard-code a release number in this file, this variable simply needs to be declared here to enable passing a value through the Makefile (`make run`, `make run_test` etc).